source/manual/world-taxonomy.html.md

```diff
@@ -0,0 +1,71 @@
+
+There are other pages on GOV.UK under the /world taxonomy that are not these pages. There is more information on this in the Relationships section of this document and a full list of taxons [here][1].
+
+Whilst world taxons are treated in the same way as the regular [taxonomy][2] on a data level, they are a separate entity and do not share the same root. They cannot be found by visiting the root of the GOV.UK taxonomy meaning that they are simultaneously part of and not part of the core taxonomy.
+
+[1]: https://content-tagger.publishing.service.gov.uk/taxons/91b8ef20-74e7-4552-880c-50e6d73c2ff9
+[2]: https://docs.publishing.service.gov.uk/manual/taxonomy.html
+
+## Editing the Topic Taxonomy
+
+The world taxonomy is managed in [content-tagger][3]. Users must have the “GDS Editor” permission in content-tagger to see the relevant pages.
+
+The topics in the taxonomy (we call them “taxons” in code) are persisted in the publishing-api as content items. For an example [see the “Afghanistan” world taxon content item][4].
+
+This means that taxons inherit the publishing-api workflow, and can be in either draft state or published.
+
+[3]: https://content-tagger.publishing.service.gov.uk/taxons
+[4]: http://www.gov.uk/api/content/world/afghanistan
+
+## Root Taxon
+
+The taxon at the root of the world taxonomy (`/world`) is rendered by whitehall because there currently isn’t a template in collections that is suitable for rendering long lists without descriptions. This means that the actual root taxon as far as content-tagger is concerned is `/world/all` which is redirected to `/world`.
+
+## Relationships
+
+The link type `parent_taxons` is used to store the relationship between taxons. A [reverse][5] link called `child_taxons` is setup through the publishing-api.
+In order to avoid having a large number of near identical taxons world taxons are based on an associated taxon structure. There are a small number of generic taxons that should be displayed on each of the individual world pages, for example [passports and emergency travel documents][6]. These are then mirrored in the world pages, for example [passports and emergency travel documents Vietnam][7], which has information specific to that country.
+
+[5]: https://github.com/alphagov/publishing-api/blob/master/lib/expansion_rules.rb#L29
+[6]: https://www.gov.uk/world/passports-and-emergency-travel-documents
+[7]: https://www.gov.uk/world/passports-and-emergency-travel-documents-vietnam
+
+## Tagging
+
+Unlike the rest of the taxonomy only content associated with [specific organisations][8] is allowed to be tagged to world taxons (DFID, FCO, UKVI, DIT, DExEU) and only [particular document types][9] (guidance, forms, detailed guides, and document collections) may be tagged to them. Presumably this is because the world pages are intended to be curation mechanisms for FCO and similar rather than content tagged to a topic.
```

Comment:
Not sure how - or even if - it fits here; or if it has any relevance, but travel organisations frequently link to these pages (or sub-pages, passports etc) for a specific country.  These are often portrayed to customers as FCO advice, and so authoritative in nature.

Reply:
When you say travel organisations, do you mean ones outside of government?

Reply:
Yes, exactly - TUI Travel Ltd for example: https://www.tui.co.uk/destinations/europe/spain/holidays-spain/travel-advice, e.g. Passports (https://www.gov.uk/foreign-travel-advice/spain/entry-requirements) and Health (https://www.gov.uk/foreign-travel-advice/spain/health).  However, they seem to have changed from using links under: https://www.gov.uk/world/spain to https://www.gov.uk/foreign-travel-advice/spain for some reason since I last saw them 😕 Not sure it's something we need to mention, just pointing out that companies do link/have linked to these.  I know as I added the URL config to the CMS for https:://www.gov.uk/world/<country> 😉

Reply:
Ah yeah, not too worried about other people linking to them. This is more to do with tagging GOV.UK pages with them on a technical level

Reply:
Yeah - makes sense and you don't need to care.  Just pointing out that people do do this.